main/activity-log.js

```diff
@@ -1,31 +1,30 @@
 'use strict'
 
 /** @typedef {import('./typings').ActivityEvent}  ActivityEvent */
-/** @typedef {import('./typings').ActivityEntry}  ActivityEntry */
+/** @typedef {import('./typings').RecordActivityOptions}  RecordActivityOptions */
```

Comment:
I think we can simplify the types by thinking about them this way: One is the actual activity (`Activity`), the other is the options passed to `.recordActivity` (`RecordActivityOptions`). To me this is clearer than `event` and `entry`, and the variable names are nicely consistent.

Reply:
We could also rename this to use `Event` instead of `Activity`, which is the more common term, but I do think here `Activity` works better, because it's clearer that we're referring to a product concept, and not a generic event.

Reply:
I like the idea to user `Activity` to describe the data structure we are storing in our log 👍🏻 

I have mixed feelings about the name `RecordActivityOptions`. In my mind, options is for optional configuration tweaking the behaviour of a function. How about something like `RecordActivityFields` or `NewActivityProps`?

Reply:
Sounds good, I tried to look up prior art on this and found that the types for node for example call arguments `options` but their type `...Args`:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3b2cb0f873d4caca3e4fc01aeaf9f8ec40852d8f/types/node/http.d.ts#L138-L139

Either would work for me 👍

Reply:
Let's use `NewActivityArgs` then.